data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+			`	Once the weather settles, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back as well.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label lonebeetle
+			`	He leaves too, and you're on your own in the ship for a couple hours. Eventually the get back, and it seems only Tugroob found something, mentioning some evidence of smuggled weapons passing through the planet.`
+			`	The three agents get back in their bunks, and say you may depart now.`
+
+
+
+mission "HI2-SF"
+	name "Investigate Stronghold of Flugbu"
+	description "Transport the Heliarch trio here so that they can investigate signs of Resistance activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Stronghold of Flugbu"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`While you doubt anyone opposing the Heliarchs would find much success in this massive compound, it seems the planet holds enough resources, ships and a valuable positioning that it was still marked for investigation.`
+			`	The three agents say you cannot go to most of the parts here they will be headed to, so you stay inside your ship. They return right as the sun is setting, and say they haven't found anything.`
+
+
+
+mission "HI2-SS"
+	name "Investigate Shifting Sand"
+	description "Transport the Heliarch trio here so that they can investigate signs of Resistance activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Shifting Sand"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`Immediately upon landing, Iplora is ready to leave your ship, going off on her own. You ask Tugroob and Kirrie about it, and they say that she is just taking a small break.`
+			`	"Value moments of solitude, the Saryd do. To a calm oasis, she will head for the day," Kirrie explains, as he also prepares to leave, claiming to know some Heliarchs stationed here who could help him look into some rumors.`
+			`	Being the last Heliarch agent in your ship, Tugroob turns to you, and asks, "Assist me in my search, would you, Captain?"`
+			choice
+				`	"Of course, where are we headed?"`
+					goto desertspider
+				`	"Sorry, I'll have to refuse this time."`
+					goto lonespider
+			label desertspider
+			`	"Reports of suspicious transactions, we have received. An outdated, abandoned warehouse, the place is. For storage of the few crops that can grow here, used, it was," they say, as you put in the coordinates they give you, and land nearby.`
```

Comment:
`	"Reports of suspicious transactions, we have received. An outdated, abandoned warehouse, the place is. For storage of the few crops that can grow here, used, it was," they say as you put in the coordinates they give you.`